engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py

```diff
@@ -47,6 +48,7 @@ class VllmRbProperties(Properties):
     device: Optional[str] = None
     preloaded_model: Optional[Any] = None
     generation_config: Optional[Any] = None
+    override_neuron_config: Optional[Dict] = None
```

Comment:
@siddvenk This one is not needed, if your `pass through` PR gets merged.

Reply:
Good to know, let's get this one merged first and then I can fixup the passthrough pr and test it with the nxdi changes here